### PR TITLE
Adding Tags Page view & tests

### PR DESCRIPTION
**Pull request by rebeccaTubman**
- Display tags feature with view test.
- Added all the many-to-many in to one class and the tags view test in
  to another class.

modified:   home/tests.py
                  home/urls.py
                  home/views.py

new file:   home/templates/home/tags.html

fix: #59

### CODE REVIEW

File: home/templates/home/tags.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            margin-top: 20px;
+        }
+        .item-in-wrapper {
+            cursor: pointer;
+            position: relative;
+            height: 200px;
+            width: 180px;
+            text-align: center;
+        }
+    </style>
```
Comment (liornoy) [inline home/templates/home/tags.html:30]: Barak recommended having all the styles in a CSS file outside of the template for improving browser-side caching.
You could put the file in the static/home/CSS directory

Comment (rebeccaTubman) [reply home/templates/home/tags.html:30]: Thank you:) I'll add a file in the directory for my page.

File: home/tests.py [hunk 2]
```diff
@@ -4,97 +4,88 @@
...
-def question_tag_test_data(question_test_data):
-    test_tag = Tag()
-    test_tag.tag_name = 'test_tag_2'
-    test_tag.save()
-    new_pair = Question_Tag()
-    new_pair.question = question_test_data
-    new_pair.tag = test_tag
-    new_pair.save()
-    return test_tag
+class TestManyToManyFeature:
```
Comment (ifireball) [inline home/tests.py:11]: this doesn't sound like the best name... names for things in your app should generally be derived from the application model element the functionality is related to

File: home/tests.py [hunk 3]
```diff
@@ -193,3 +184,24 @@ def test_response_context(self, response):
...
+
+class TestTagsPage:
+    @pytest.mark.django_db
+    def test_tags_page_url(self, tags_response):
+        assert tags_response.status_code == 200
+
+    @pytest.mark.django_db
+    def test_tags_page_context(self, tags_response):
+        excepted = '<QuerySet [<Tag: 5th_Grade>, <Tag: Bagrut_Exam>, <Tag: Hebrew>, <Tag: Pitagoras>]>'
+        assert str(tags_response.context['tags']) == excepted
```
Comment (ifireball) [inline home/tests.py:197]: its usually a bad idea to use this kind of single-string comparison when you have a more complex data structure. Consider that you might change `__repr__` for various reasons in the future and break the test.

What you should do instead it break down the elements ou want to test - wanna test the type of the object youve got? Use `isinstance`. Wanna test that you got a list of a particular data items? Convert is to a list of simpler objects, like I did in the testing examples I wrote for the message board.

**Review by ifireball — APPROVED** [review]
Approving this in the interest of time, because it does seem like you have sufficient test coverage for the feature you've introduced. 

Please see my comments about various elements in the code and apply fixes later.

**Pull request merged**